pkg/compliance/checks/builder.go

```diff
@@ -102,6 +102,8 @@ func (b *builder) checkFromRule(meta *compliance.SuiteMeta, ruleID string, ruleS
 		return b.dockerCheck(meta, ruleID, ruleScope, resource.Docker)
 	case resource.Process != nil:
 		return newProcessCheck(b.baseCheck(ruleID, "process", ruleScope, meta), resource.Process)
+	case resource.Command != nil:
+		return newCommandCheck(b.baseCheck(ruleID, "command", ruleScope, meta), resource.Command)
```

Comment:
I like this style of initializing - I should probably decouple other checks I introduced from the builder like this.